wgsl/index.bs

```diff
@@ -1906,6 +1906,8 @@ memory reference</dfn> is produced.
     * store the value to any [=memory locations|memory location(s)=] of the
         [[WebGPU#buffers|WebGPU buffer]] bound to the [=originating variable=]
     * not be executed
+[[#atomic-rmw|Read-modify-write atomics]] must load and store from the same
+[=memory locations|memory locations=] if they access memory.
```

Comment:
Ah, so this "if" is a run-time property, not compile time. I see.
It would be neat to have that written a little more specific, i.e. "unless the instructions end up not accessing memory due to out-of-bounds addressing".

Reply:
Oops, right, I lost that part in the update.